gr-blocks/grc/blocks_tag_gate.xml

```diff
@@ -54,6 +57,13 @@
       <key>False</key>
     </option>
   </param>
+
```

Comment:
Could we also take this opportunity to change the name of the propagate tags parameter from `<name>Propagate_tags</name>` to `<name>Propagate Tags</name>` on line 47? The old name seems kinda wonky.